README.md

```diff
@@ -27,21 +27,12 @@ faster than other packages according to public benchmarks.
 
 ## Installation
 
-We assume that you have Tarantool version 1.6 and a modern Linux or BSD
+We assume that you have Tarantool version 1.6+ and a modern Linux, BSD or macOS
```

Comment:
I suppose it is a question to initial authors?
The current version of CI doesn't contain a macOS env, so it is undefined. And it is out of scope of task with documentation.

Reply:
macOS is removed. Without regular testing, we cannot guarantee that `go-tarantool` works on it.

```diff
--- a/README.md
+++ b/README.md
@@ -27,7 +27,7 @@ faster than other packages according to public benchmarks.
 
 ## Installation
 
-We assume that you have Tarantool version 1.6+ and a modern Linux, BSD or macOS
+We assume that you have Tarantool version 1.6+ and a modern Linux or BSD
 operating system.
 
 You will need a current version of `go`, version 1.13 or later (use `go version`
```

Reply:
> macOS is removed. Without regular testing, we cannot guarantee that go-tarantool works on it.

I think we may try to ask Presale Team, since there are some people that uses go-tarantool. Maybe they can say that everything is alright

Reply:
> I think we may try to ask Presale Team, since there are some people that uses go-tarantool. Maybe they can say that everything is alright

I would not rely on it.
Submitted a new issue about it - #157.